src/lib/core/CHIPError.h

```diff
@@ -618,7 +618,14 @@ using CHIP_ERROR = ::chip::ChipError;
  */
 #define CHIP_ERROR_INVALID_TLV_CHAR_STRING                     CHIP_CORE_ERROR(0x15)
 
-// AVAILABLE: 0x16
+/**
+ *  @def CHIP_ERROR_LIT_SUBSCRIBE_INACTIVE_TIMEOUT
+ *
+ *  @brief
+ *    Subscription timeout caused by LIT ICD device inactive mode
+ *
+ */
+#define CHIP_ERROR_LIT_SUBSCRIBE_INACTIVE_TIMEOUT                     CHIP_CORE_ERROR(0x16)
```

Comment:
```suggestion
#define CHIP_ERROR_LIT_SUBSCRIBE_INACTIVE_TIMEOUT              CHIP_CORE_ERROR(0x16)
```
that's how all the other ones are lined up.